src/Repository/CommitteeFeedItemRepository.php

```diff
@@ -33,9 +33,8 @@ private function findMostRecentFeedItem(string $type, string $committeeUuid = nu
     {
         $qb = $this
             ->createQueryBuilder('i')
-            ->select('i, a, e')
+            ->select('i, a')
             ->leftJoin('i.author', 'a')
-            ->leftJoin('i.event', 'e')
```

Comment:
Why do you remove the left join? You can keep it and put a WITH. Here, it would try to load the event lazily.